dcs-packaging-tool/dcs-packaging-tool-impl/src/test/java/org/dataconservancy/packaging/tool/impl/GeneralPackageDescriptionCreatorTest.java

```diff
@@ -370,6 +371,7 @@ public void simLinkCycleTest() throws Exception {
             Files.createSymbolicLink(link, subdir.toPath());
         } catch (UnsupportedOperationException e) {
             /* Nothing we can do if the system doesn't support symlinks */
+            FileUtils.deleteDirectory(tempDir);
```

Comment:
My thought would be to just make the names unique, use the method names as the names of the file or something similar. Or just any unique id as in most cases the file names don't matter for the test. The problem with relying on file deletes is that they won't always work on Windows. In some tests the files will be locked until after the test executes so the delete won't execute.